client/blocks/plugins-update-manager/schedule-form.tsx

```diff
@@ -39,11 +48,17 @@ import './schedule-form.scss';
 interface Props {
 	scheduleForEdit?: ScheduleUpdates;
 	onSyncSuccess?: () => void;
+	onCreateMonitorSuccess?: () => void;
 }
 export const ScheduleForm = ( props: Props ) => {
+	const CRON_CHECK_INTERVAL = 5;
 	const moment = useLocalizedMoment();
 	const siteSlug = useSiteSlug();
-	const { scheduleForEdit, onSyncSuccess } = props;
+	const siteId = useSelector( ( state ) => getSiteId( state, siteSlug ) );
```

Comment:
The form file becomes so crowded.
Can we move getters of `siteId` and `siteUrl` to the hooks? Like we did for the siteSlug?

What do you think? Does it make sense?

Reply:
Of course. Moved to hook.